docs/quick-start.md

```diff
@@ -0,0 +1,130 @@
+- [Quick Start Guide for Ceph-CSI-Operator](#quick-start-guide-for-ceph-csi-operator)
+  - [1. Prerequisites](#1-prerequisites)
+  - [2. Install the Ceph-CSI Operator](#2-install-the-ceph-csi-operator)
+    - [3. Create OperatorConfig CR](#3-create-operatorconfig-cr)
+    - [4. Deploy Ceph-CSI Drivers](#4-deploy-ceph-csi-drivers)
+      - [4.1 Deploy the RBD Driver](#41-deploy-the-rbd-driver)
+      - [4.2 Deploy the CephFS Driver](#42-deploy-the-cephfs-driver)
+    - [4.3 Deploy the Ceph-NFS Driver](#43-deploy-the-ceph-nfs-driver)
+  - [5. Verify Installation](#5-verify-installation)
+  - [6. Clean Up Resources](#6-clean-up-resources)
+
+# Quick Start Guide for Ceph-CSI-Operator
+
+## 1. Prerequisites
+
+Before deploying the Ceph-CSI-Operator, ensure the following requirements are met:
+
+- A Kubernetes cluster ([supported version](https://kubernetes.io/releases/) recommended)
+- Ceph cluster ([supported version](https://docs.ceph.com/en/latest/releases/) recommended)
+- `kubectl` CLI installed
+
+**Note:** In this guide, we will use minimal configurations to deploy the Ceph-CSI-Operator and drivers. You can customize the configurations as per your requirements.
+
+## 2. Install the Ceph-CSI Operator
+
+```console
+kubectl create -f deploy/all-in-one/install.yaml
+```
+
+verify the installation:
+
+```bash
+kubectl get pods -n ceph-csi-operator-system
+NAME                                                    READY   STATUS    RESTARTS   AGE
+ceph-csi-operator-controller-manager-67d45fd9ff-zgst7   2/2     Running   0          40s
+```
+
+### 3. Create OperatorConfig CR
+
+Create an OperatorConfig CR to configure the Ceph-CSI Operator:
+
+```console
+echo '
+apiVersion: csi.ceph.io/v1alpha1
+kind: OperatorConfig
+metadata:
+  name: ceph-csi-operator-config
+  namespace: ceph-csi-operator-system
+' | kubectl create -f -
```

Comment:
If we are going for minimal installation, why do we need an operator config instance?

Reply:
it was added to deploy the CR with minimal values and no values and the user can update it later as per requirement. Removed it now